components/formats-gpl/src/loci/formats/in/LiFlimReader.java

```diff
@@ -376,6 +421,11 @@ private void initCoreMetadata() throws FormatException {
     ms.sizeZ = Integer.parseInt(zLen) * sizeF;
     ms.sizeC = Integer.parseInt(channels);
     ms.sizeT = Integer.parseInt(timestamps) * sizeP;
+    
+    /** Check for DarkImage, if "1" increment Timestamps with 1 such that DarkImage is appended to the stream  */
+    if (DarkImage.equals(new String("1"))) {
```

Comment:
See https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-test-folder/51263/console this causes a NPE for  sample images where `DarkImage` is not defined so the nullity needs to be checked

Reply:
Hello Sébastian, fixed it and added some more fixes.